### DISCUSSION TITLE
AI deck images are dark

### DISCUSSION THREAD

**Discussion by noahzn** [bitcraze/discussions#451] [Q&A] [answered]
I replaced the RGB camera with a grayscale camera. I ran `fpv.py` and saved some images. I found that these images are very dark. Is there a way that I can control the brightness of images? I would like to know more about the camera adjustment. Thanks.
![668bbaa05905e1d87abfb4f656ab37d](https://user-images.githubusercontent.com/8244850/205311080-6bffe29c-c06c-4b4a-91be-36ae662a7900.jpg)

**Comment by noahzn** [comment, marked as answer]
OK, I think this is caused by the automatic exposure compensation of the camera. The room has two very bright lights, so the image is dark.

**Reply by knmcguire** [reply]
That could be it! It should be able to set some registries to try to compensate for this. But ofcourse there is a limitation of what this small camera can do.

**Reply by noahzn** [reply]
@knmcguire do you know how can I adjust its brightness?
